packages/client/src/impl/WriteApiImpl.ts

```diff
@@ -111,6 +119,6 @@ export default class WriteApiImpl implements WriteApi {
   }
 
   async close(): Promise<void> {
-    this.closed = true
+    this._closed = true
   }
```

Comment:
Is `close()` necessary if it does nothing?

Reply:
closing makes next write to throw. This behaviour is good to have because it normalizes behaviour of write against query. Also it will be ready if there is change in underlying implementation (requiring close) for write in future